README.md

```diff
@@ -74,7 +74,7 @@ albibong
 2. Run the Backend inside `src`
 
 ```
-cd src && python albibong
+cd src && python -m albibong
```

Comment:
For developers, it would be nice to have additional instructions on how to set the project to run it with debug mode eg in VSCode. The issue is that I need on iOs to run it with sudo permission due to some conflicts with socket tracker. `sudo python -m albibong` works but in VSCode is not possible to start project with it ;(

Reply:
> how to set the project to run it with debug mode

I'm actually writing all this using VSCode and I use the debugger as well. Should I include the launch.json to the repo?

> need on iOs to run it with sudo permission

Can you elaborate more on this part? I don't think I need to run the program with sudo permission... `python -m albibong` works just fine.

Reply:
This error I got when I try to run it without sudo or from VSCode
```
Traceback (most recent call last):
  File "/Users/username/.pyenv/versions/3.9.18/lib/python3.9/site-packages/scapy/arch/bpf/core.py", line 147, in get_dev_bpf
    fd = os.open("/dev/bpf%i" % bpf, os.O_RDWR)
PermissionError: [Errno 13] Permission denied: '/dev/bpf0'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/username/.pyenv/versions/3.9.18/lib/python3.9/threading.py", line 980, in _bootstrap_inner
    self.run()
  File "/Users/username/.pyenv/versions/3.9.18/lib/python3.9/threading.py", line 917, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/username/.pyenv/versions/3.9.18/lib/python3.9/site-packages/scapy/sendrecv.py", line 1171, in _run
    sniff_sockets[_RL2(iface)(type=ETH_P_ALL, iface=iface,
  File "/Users/username/.pyenv/versions/3.9.18/lib/python3.9/site-packages/scapy/arch/bpf/supersocket.py", line 331, in __init__
    super(L2bpfListenSocket, self).__init__(*args, **kwargs)
  File "/Users/username/.pyenv/versions/3.9.18/lib/python3.9/site-packages/scapy/arch/bpf/supersocket.py", line 108, in __init__
    (self.ins, self.dev_bpf) = get_dev_bpf()
  File "/Users/username/.pyenv/versions/3.9.18/lib/python3.9/site-packages/scapy/arch/bpf/core.py", line 151, in get_dev_bpf
    raise Scapy_Exception((
scapy.error.Scapy_Exception: Permission denied: could not open /dev/bpf0. Make sure to be running Scapy as root ! (sudo)
```

And this is my launch.json
```
{
  "version": "0.2.0",
  "configurations": [
    {
      "name": "Python: Albion",
      "type": "python",
      "request": "launch",
      "module": "albibong",
      "cwd": "${workspaceFolder}/src",
      "console": "integratedTerminal",
      "pythonPath": "/Users/username/.pyenv/versions/3.9.18/bin/python"
    }
  ]
}

```

Reply:
maybe the case is on how I was install the python with pyenv IDK